src/app/core/entity-components/entity-form/invalid-form-field.error.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Custom error indicating details of a failed form validation.
+ */
+export class InvalidFormFieldError extends Error {
+  constructor(message: string) {
+    super(message);
+    Object.setPrototypeOf(this, InvalidFormFieldError.prototype);
```

Comment:
why is this necessary?

Reply:
because stackoverflow :wink: 

actually not necessary here but would be needed if we add methods to the custom class (https://stackoverflow.com/a/41429145/1473411)